src/timescale.rs

```diff
@@ -99,6 +105,15 @@ impl TimeScale {
     pub const fn is_gnss(&self) -> bool {
         matches!(self, Self::GPST | Self::GST | Self::BDT)
     }
+    /// Returns Reference Epoch (t(0)) for given timescale
+    pub(crate) const fn ref_epoch(&self) -> Epoch {
+        match self {
+            Self::GPST => GPST_REF_EPOCH,
+            Self::GST => GST_REF_EPOCH,
+            Self::BDT => BDT_REF_EPOCH,
+            _ => J1900_REF_EPOCH,
```

Comment:
```rust
_ => J1900_REF_EPOCH
```

is that true ?

Reply:
Yes, I'm pretty sure. To check, we can change the duration calculations for these time scales to use this `ref_epoch` const function: if the tests fail, then something is wrong.

Ah, actually, I think that ET and TDB need their own ref epoch of J2000 in their respective time scale.

Reply:
> Ah, actually, I think that ET and TDB need their own ref epoch of J2000 in their respective time scale

Yep, my concern was about these two, which I am not familiar with, and I saw you quite often refer to J2000 when dealing with them